\documentclass[a4paper,12pt]{extarticle}
\usepackage{amsmath}
\usepackage{tikz}
\usepackage{blindtext}
\usepackage{enumitem}
\usepackage{xcolor}
\usepackage{epigraph}
\usepackage{lipsum}
\usepackage{graphicx}
\graphicspath{{images/}}
\usepackage{hyperref}
\hypersetup{colorlinks = true, citecolor = blue, linkcolor = blue, urlcolor = blue}

\renewcommand\epigraphflush{flushright}
\renewcommand\epigraphsize{\normalsize}
\setlength\epigraphwidth{1\textwidth}


\definecolor{titlepagecolor}{rgb}{0.63,0.36,0.94}

{%
\centering
\DeclareFixedFont{\titlefont}{T1}{ppl}{b}{it}{0.5in}
}
    
\newcommand\titlepagedecoration{%
\begin{tikzpicture}[remember picture,overlay,shorten >= -10pt]

\coordinate (aux1) at ([yshift=-15pt]current page.north east);
\coordinate (aux2) at ([yshift=-390pt]current page.north east);
\coordinate (aux3) at ([xshift=-4.5cm]current page.north east);
\coordinate (aux4) at ([yshift=-150pt]current page.north east);


\begin{scope}[titlepagecolor!40,line width=18pt,rounded corners=5pt]
\draw
  (aux1) -- coordinate (a)
  ++(225:5) --
  ++(-45:5.1) coordinate (b);
\draw[shorten <= -10pt]
  (aux3) --
  (a) --
  (aux1);
\draw[opacity=0.6,titlepagecolor,shorten <= -10pt]
  (b) --
  ++(225:2.2) --
  ++(-45:2.2);
\end{scope}
\draw[titlepagecolor,line width=12pt,rounded corners=5pt,shorten <= -10pt]
  (aux4) --
  ++(225:0.8) --
  ++(-45:0.8);
\begin{scope}[titlepagecolor!70,line width=8pt,rounded corners=5pt]
\draw[shorten <= -10pt]
  (aux2) --
  ++(225:3) coordinate[pos=.45] (c) --
  ++(-45:3.1);
\draw
  (aux2) --
  (c) --
  ++(135:2.5) --
  ++(45:2.5) --
  ++(-45:2.5) coordinate[pos=0.3] (d);   
\draw 
  (d) -- +(45:1);
\end{scope}
\end{tikzpicture}%
}


\begin{document}
\begin{titlepage}

\noindent
{%
\centering
\titlefont{\color{black} PRIME EDITING} \par
}



\begin{figure}[h]
\includegraphics[width=\textwidth]{name.jpeg}
\centering
\end{figure}

\begin{figure}[h]
\includegraphics[scale=0.9]{lg.jpeg}
\centering
\end{figure}

\begin{figure}[h]
\includegraphics[scale=0.621]{sit.jpeg}
\centering
\end{figure}





\vfill
\noindent



\titlepagedecoration
\pagebreak

\begin{figure}[h]
\includegraphics[scale=1]{ack.jpeg}
\centering
\end{figure}
 

\end{titlepage}

\clearpage

\tableofcontents

\clearpage

\section{Introduction}

Prime editing is a ‘search-and-replace’ genome editing technology in molecular biology by which the genome of living organisms may be modified. The technology directly writes new genetic information into a targeted DNA site. It uses a fusion protein, consisting of a catalytically impaired Cas9 endonuclease fused to an engineered reverse transcriptase enzyme, and a prime editing guide RNA (pegRNA), capable of identifying the target site and providing the new genetic information to replace the target DNA nucleotides. It mediates targeted insertions, deletions, and base-to-base conversions witho- ut the need for double strand breaks (DSBs) or donor DNA templates.\\

Prime editing was developed in the lab of David R. Liu at the Broad Institute and disclosed in Anzalone et al. 2019. Since then prime editing and the research that produced it have received widespread scientific acclaim, being called "revolutionary" and an important part of the future of editing. The technology is an early-stage, experimental genome editing method that has received mainstream press attention due to its potential uses in medical genetics. It utilizes similar methodologies to precursor genome editing techno- logies, including CRISPR/Cas9 and base editors. Prime editing has been used to treat some animal models of genetic disease.\\

\section{Components}

Prime editing involves three major components:\\

\begin{itemize}

\item A \textbf{prime editing guide RNA (pegRNA)}, capable of (i) identifying the target nucleotide sequence to be edited, and (ii) encoding new genetic information that replaces the targeted sequence. The pegRNA consists of an extended single guide RNA (sgRNA) containing a primer binding site (PBS) and a reverse transcriptase (RT) template sequence. During genome editing, the primer binding site allows the 3’ end of the nicked DNA strand to hybridize to the pegRNA, while the RT template serves as a template for the synthesis of edited genetic information.

\item A fusion protein consisting of a Cas9 H840A nickase fused to a Moloney Murine Leukemia Virus (M-MLV) reverse transcriptase.

\begin{enumerate}

\item \textbf{Cas9 H840A nickase}: the Cas9 enzyme contains two nuclease domains that can cleave DNA sequences, a RuvC domain that cleaves the non-target strand and a HNH domain that cleaves the target strand. The introduction of a H840A substitution in Cas9, through which the 840º amino acid histidine is replaced by an alanine, inactivates the HNH domain. With only the RuvC functioning domain, the catalytically impaired Cas9 introduces a single strand nick, hence the name nickase.

\item \textbf{M-MLV reverse transcriptase}: an enzyme that synthesizes DNA from a single-stranded RNA template.

\end{enumerate}

\item A single guide RNA (sgRNA) that directs the Cas9 H840A nickase portion of the fusion protein to nick the non-edited DNA strand.

\end{itemize}

\begin{figure}[h]
\includegraphics[scale=0.35]{components.jpeg}
\centering
\caption{Components of prime editing}
\centering
\end{figure}

\section{Mechanism}

Genomic editing takes place by transfecting cells with the pegRNA and the fusion protein. Transfection is often accomplished by introducing vectors into a cell.\\

Once internalized, the fusion protein nicks the target DNA sequence, exposing a 3’-hydroxyl group that can be used to initiate (prime) the reverse transcription of the RT template portion of the pegRNA.\\

This results in a branched intermediate that contains two DNA flaps: a 3’ flap that contains the newly synthesized (edited) sequence, and a 5’ flap that contains the dispensable, unedited DNA sequence.\\

The 5’ flap is then cleaved by structure-specific endonucleases or 5’ exonuc- leases. This process allows 3’ flap ligation, and creates a heteroduplex DNA composed of one edited strand and one unedited strand. The reannealed double stranded DNA contains nucleotide mismatches at the location where editing took place.\\

In order to correct the mismatches, the cells exploit the intrinsic mismatch repair mechanism, with two possible outcomes: (i) the information in the edited strand is copied into the complementary strand, permanently installing the edit; (ii) the original nucleotides are re-incorporated into the edited strand, excluding the edit.\\

\begin{figure}[h]
\includegraphics[width=\textwidth]{method.jpeg}
\centering
\caption{Prime editing mechanism}
\centering
\end{figure}

\section{Development process}

During the development of this technology, several modifications were done to the components, in order to increase its effectiveness.\\

\subsection{Prime editor 1}

In the first system, a wild-type Moloney Murine Leukemia Virus (M-MLV) reverse transcriptase was fused to the Cas9 H840A nickase C-terminus. Detec- table editing efficiencies were observed.\\

\subsection{Prime editor 2}

In order to enhance DNA-RNA affinity, enzyme processivity, and thermostabi- lity, five amino acid substitutions were incorporated into the M-MLV reverse transcriptase. The mutant M-MLV RT was then incorporated into PE1 to give rise to (Cas9 (H840A)-M-MLV RT). Efficiency improvement was observed over PE1.\\

\subsection{Prime editor 3}

Despite its increased efficacy, the edit inserted by PE2 might still be removed due to DNA mismatch repair of the edited strand. To avoid this problem during DNA heteroduplex resolution, an additional single guide RNA (sgRN- A) is introduced. This sgRNA is designed to match the edited sequence introduc- ed by the pegRNA, but not the original allele. It directs the Cas9 nickase portion of the fusion protein to nick the unedited strand at a nearby site, opposite to the original nick. Nicking the non-edited strand causes the cell's natural repair system to copy the information in the edited strand to the complementary strand, permanently installing the edit.[1] However, there are drawbacks to this system as nicking the unaltered strand can lead to additional undesired indels.

\subsection{Prime editor 4}

Prime editor 4 utilizes the same machinery as PE2, but also includes a plasmid that encodes for dominant negative MLH1. Dominant negative MLH1 is able to essentially knock out endogenous MLH1 by inhibition, thereby reducing cellular MMR response and increasing prime editing efficiency.

\begin{figure}[h]
\includegraphics[scale=0.3]{pe3.jpeg}
\centering
\caption{PE3 system for prime editing}
\centering
\end{figure}

\subsection{Prime editor 5}

Prime editor 5 utilizes the same machinery as PE3, but also includes a plasmid that encodes for dominant negative MLH1. Like PE4, this allows for a knockdown of endogenous MMR response, increasing the efficiency of prime editing.

\subsection{Nuclease Prime Editor}

Nuclease Prime Editor uses Cas9 nuclease instead of Cas9(H840A) nickase. Unlike prime editor 3 (PE3) that requires dual-nick at both DNA strands to induce efficient prime editing, Nuclease Prime Editor requires only a single pegRNA since the single-gRNA already creates double-strand break instead of single-strand nick.

\subsection{Twin Prime Editing}

The "twin prime editing" (twinPE) mechanism reported in 2021 allows editing large sequences of DNA – sequences as large as genes – which addresses the method's key drawback. It uses a prime editor protein and two prime editing guide RNAs.

\section{Advantages}

The prime editing tool offers advantages over traditional gene editing technolo- gies. CRISPR/Cas9 edits rely on non-homologous end joining (NHEJ) or homology-directed repair (HDR) to fix DNA breaks, while the prime editing system employs DNA mismatch repair. This is an important feature of this technology given that DNA repair mechanisms such as NHEJ and HDR, generate unwanted, random insertions or deletions (INDELs) byproducts which complicate the retrieval of cells carrying the correct edit.\\

The prime system introduces single-stranded DNA breaks instead of the double-stranded DNA breaks observed in other editing tools, such as base editors. Collectively, base editing and prime editing offer complementary strengths and weaknesses for making targeted transition mutations.\\

Base editors offer higher editing efficiency and fewer INDEL byproducts if the desired edit is a transition point mutation and a PAM sequence exists roughly 15 bases from the target site. However, because the prime editing technology does require a precisely positioned PAM sequence to target a nucleotide sequence, it offers more flexibility and editing precision.\\

 Remarkably, prime editors allow all types of substitutions, transitions and transversions to be inserted into the target sequence.\\
 
 Because the prime system involves three separate DNA binding events (between (i) the guide sequence and the target DNA, (ii) the primer binding site and the target DNA, and (iii) the 3’ end of the nicked DNA strand and the pegRNA), it has been suggested to have fewer undesirable off-target effects than CRISPR/Cas9.\\
 
\section{Limitations}

There is considerable interest in applying gene-editing methods to the treatm- ent of diseases with a genetic component. However, there are multiple challenges associated with this approach. An effective treatment would require editing of a large number of target cells, which in turn would require an effective method of delivery and a great level of tissue specificity.\\

As of 2021, prime editing looks promising for relatively small genetic alterations, but more research needs to be conducted to evaluate whether the technology is efficient in making larger alterations, such as targeted insertions and deletions. \\

Larger genetic alterations would require a longer RT template, which could hinder the efficient delivery of pegRNA to target cells. Furthermore, a pegRNA containing a long RT template could become vulnerable to damage caused by cellular enzymes.\\

Prime editing in plants suffers from low efficiency ranging from zero to a few percent and needs significant improvement.\\

Some of these limitations have been mitigated by recent improvements to the prime editors, including motifs that protect pegRNAs from degradation.\\

\section{Delivery method}

Base editors used for prime editing require delivery of both a protein and RNA molecule into living cells. Introducing exogenous gene editing technolog- ies into living organisms is a significant challenge. One potential way to introduce a base editor into animals and plants is to package the base editor into a viral capsid. The target organism can then be transduced by the virus to synthesize the base editor in vivo. Common laboratory vectors of transduction such as lentivirus cause immune responses in humans, so proposed human therapies often centered around adeno-associated virus (AAV) because AAV infections are largely asymptomatic.

\section{Conclusion}

PE is among recently introduced technologies which allow the transfer of single-nucleotide polymorphisms (SNPs) from one individual crop plant to another. PE is precise enough to be used to recreate an arbitrary SNP in an arbitrary target, including deletions, insertions, and all 12 point mutations without also needing to perform a double-stranded break or carry a donating template.\\

Prime editors may be used in gene drives. A prime editor may be incorporated into the Cleaver half of a Cleave and Rescue/ClvR system. In this case it is not meant to perform a precise alteration but instead to merely disrupt.\\

Although additional research is required to improve the efficiency of prime editing, the technology offers promising scientific improvements over other gene editing tools. The prime editing technology has the potential to correct the vast majority of pathogenic alleles that cause genetic diseases, as it can repair insertions, deletions, and nucleotide substitutions.

\pagebreak

\section{References}

Search-and-replace genome editing without double-strand breaks or donor DNA Andrew V. Anzalone, Peyton B. Randolph, Jessie R. Davis, Alexander A. Sousa, Luke W. Koblan, Jonathan M. Levy, Peter J. Chen, Christopher Wilson, Gregory A. Newby, Aditya Raguram and David R. Liu\\

Comprehensive analysis of prime editing outcomes in human embryonic stem cells 
Omer Habib,  Gizem Habib,  Gue-Ho Hwang,  Sangsu Ba\\

Wikipedia Page on Prime Editing\\

Nelson, James (October 4, 2021). "Engineered pegRNAs improve prime editing efficiency"\\

 "Dual-AAV delivering split prime editor system for in vivo genome editing" Zhi, Shengyao; Chen, Yuxi; Wu, Guanglan\\
 
 Dicorato, Allessandra. "New prime editing system inserts entire genes in human cells"\\
 
https://www.sigmaaldrich.com/IN/en/technical-documents/technical-art icle /genomics/advanced-gene-editing/prime-editing




\end{document}